Tests/SonarScanner.MSBuild.Common.Test/ILoggerTests.cs

```diff
@@ -434,6 +434,16 @@ public void ConsoleLogger_WriteTelemetryMessages_DifferentValueTypes()
              Path.Combine(outputDir, FileConstants.TelemetryFileName), contents);
     }
 
+    [TestMethod]
+    public void ConsoleLogger_WriteTelemetryMessages_IOException_DoesNotThrow()
+    {
+        var logger = new ConsoleLogger(false);
```

Comment:
I will assert the message in the next PR.
There's no infra to do it here - I would like to merge this so we can move on.

Reply:
In the same file we used
```cs
        var recorder = new OutputRecorder();
        var logger = ConsoleLogger.CreateLoggerForTesting(true, recorder, fileWrapper);
```
This looks simple enough for me.